src/geometry/manifold/mfderiv.lean

```diff
@@ -543,7 +543,7 @@ omit Is I's
 /-! ### Deriving continuity from differentiability on manifolds -/
 
 theorem has_mfderiv_within_at.continuous_within_at
-  (h : mdifferentiable_within_at I I' f s x) : continuous_within_at f s x :=
+  (h : has_mfderiv_within_at I I' f s x f') : continuous_within_at f s x :=
```

Comment:
This is actually a different assumption, right?  Because the `f'` should not have been in scope before.

Reply:
Oh I see .. this is a copy-paste error as you said.

Reply:
@hrmacbeth  I really don't know this part of the library at all, so I pretty much just tried random things that matched the name until it compiled! Does it seem ok to you?

Reply:
Yes, sorry, I believe you have done exactly right.